src/sdk/namespace_cache.js

```diff
@@ -289,11 +306,13 @@ class NamespaceCache {
                     'size',
                     'etag',
                     'xattr',
-                    'complete_upload'
+                    'complete_upload',
+                    'last_modified_time',
```

Comment:
make a comment that we don't really expect it from params and will override anyway but is here for the type info.

Reply:
ok